src/components/molecules/Chatbox/components/ChatboxOptionsControls/ChatboxOptionsControls.scss

```diff
@@ -0,0 +1,59 @@
+@import "scss/constants";
+
+.ChatboxOptionsControls {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: $spacing--sm $spacing--md 0 $spacing--md;
+
+  &__dropdown.dropup {
+    display: flex;
+    justify-content: flex-end;
+
+    .dropdown-toggle {
+      background: none;
+      padding: 0;
+      border-radius: 0;
+      text-decoration: none;
+
+      font-size: $font-size--sm;
+      font-weight: $font-weight--400;
+      color: opaque-white(0.8);
+      border-bottom: thin solid opaque-white(0.8);
+
+      &:focus {
+        box-shadow: none;
+      }
+
+      &:hover:enabled {
+        background-image: none;
+        transform: none;
+        color: opaque-white(0.9);
+        border-bottom: thin solid opaque-white(0.9);
+      }
+
+      &::after {
+        display: none;
+      }
+    }
+
+    .dropdown-menu {
+      background-color: $primary;
+      border-radius: 22px;
```

Comment:
from constants

Reply:
fixed in f2fa24aadcae7547799e463bd593e8a7a2cf05b5